src/Veldrid/ValidationHelpers.cs

```diff
@@ -8,6 +9,8 @@ internal static class ValidationHelpers
         internal static void ValidateResourceSet(GraphicsDevice gd, ref ResourceSetDescription description)
         {
 #if VALIDATE_USAGE
+            if (description.Layout is null)
+                throw new NullReferenceException("There is no associated ResourceLayout in the provided ResourceSetDescription");
```

Comment:
`NullReferenceException` isn't right here, I would just make this a `VeldridException`. Veldrid doesn't really throw a large variety of different exceptions.